hax/treemakers/corrections.py

```diff
@@ -87,4 +168,19 @@ def extract_data(self, event):
         result['cs2_top'] = s2.area * s2.area_fraction_top * s2_top_correction
         result['cs2_bottom'] = s2.area * (1.0 - s2.area_fraction_top) * s2_bottom_correction
 
```

Comment:
cs2 is in the corrections minitree only. Blinding is changed so that it only runs if corrections loaded.

I'm not entirely sure this is the best but we don't want cs2 in basics. It will not be the same as cs2 from corrections so this is too confusing.